FreeRTOS_Sockets.c

```diff
@@ -1965,21 +1965,17 @@ BaseType_t FreeRTOS_setsockopt( Socket_t xSocket,
                            break; /* will return -pdFREERTOS_ERRNO_EINVAL */
                        }
 
-                       if( ( pxTCP->txStream != NULL ) || ( pxTCP->rxStream != NULL ) )
-                       {
-                           FreeRTOS_debug_printf( ( "Set SO_WIN_PROP: buffer already created\n" ) );
-                           break; /* will return -pdFREERTOS_ERRNO_EINVAL */
-                       }
```

Comment:
Also good: `prvSockopt_so_buffer()` will check if the stream buffers have already been created.